docs/references/general/rawacf.md

```diff
@@ -84,11 +77,10 @@ RAWACF files contain a record that contains scalar and vector fields.
 | *rawacf.revision.minor* | *None*   | int       | Minor version number of the RAWACF algorithm                                            |
 | *thr*                   | *None*   | float     | Threshold factor                                                                        |
 
-
-### Vectors 
-
 !!! Note
-    *slist* contains the range gates that obtained data points during the integration period of the beam. 
+    The original intended usage of the `channel` parameter was to identify the data from the two separate frequency channels of the stereo radars (e.g. Hankasalmi). However, this parameter has sometimes been used for non-stereo radars when data from two different frequencies have been recorded in a single file.
```

Comment:
same as above (especially since Stereo radars are known for using different beams and frequencies on different channels)